src/fidesops/schemas/saas/saas_config.py

```diff
@@ -192,7 +192,35 @@ class SaaSType(Enum):
     custom = "custom"
 
 
-class SaaSConfig(BaseModel):
+class SaaSConfigBase(BaseModel):
+    """
+    Used to store base info for a saas config
+    """
+
+    fides_key: FidesOpsKey
+    name: str
+    type: SaaSType
```

Comment:
I have used this name before and it caused errors because `type` is a Python built in function. You can change the variable name, and still allow it to be named `type` in the toml/yaml files.

```suggestion
from pydantic import Field

connection_type: SaasType = Field(..., alias="type")  # ... makes it required
```

Reply:
so I'm actually going to keep this as it was before, since it wasn't causing any errs. Updating this requires a bunch of other changes, so I want to keep this PR scoped down for now